runtime/onert/core/src/backend/controlflow/KernelGenerator.cc

```diff
@@ -135,13 +163,31 @@ std::shared_ptr<backend::ITensor> KernelGenerator::getTensor(const ir::OperandIn
     auto tensor = tensor_builder->tensorAt(index);
     if (tensor)
     {
+      assert(ret == nullptr);
       ret = tensor;
     }
   }
   assert(ret != nullptr);
   return ret;
 }
 
+std::shared_ptr<backend::ITensorBuilder>
+KernelGenerator::getTensorBuilder(const ir::OperandIndex &index)
+{
+  std::shared_ptr<backend::ITensorBuilder> ret;
+  for (auto tensor_builder : _tensor_builder_set)
+  {
+    auto tensor = tensor_builder->tensorAt(index);
+    if (tensor)
+    {
+      assert(ret == nullptr);
+      ret = tensor_builder;
```

Comment:
how about `break;` after this line?

Reply:
Okay, I will do that. And Adding `break`, the above assertion should be removed.